pom.xml

```diff
@@ -190,17 +190,67 @@
                 <additionalparam>-Xdoclint:none</additionalparam>
             </properties>
         </profile>
+        <profile>
+            <id>internal-repository</id>
```

Comment:
I would prefer calling the profile `jfrog-repository` instead of `internal-repository` because the code is open source. Not a big deal, just something to consider.

Reply:
I actually rename it from jfrog-repository as we can migrate to internal repository of github instead of artifactory so I try to keep name agnostic